assets/src/domain/eventEditor/ui/datetimes/hooks/useEditDatetimeModal.ts

```diff
@@ -4,24 +4,42 @@ import { FormProps } from 'react-final-form';
 
 import useDateFormConfig from '../dateForm/useDateFormConfig';
 import { useFormModal, FormModal, ModalClose } from '@application/ui/layout/formModal';
-import { useDatetimeMutator, UpdateDatetimeInput } from '@edtrServices/apollo/mutations';
+import {
+	useDatetimeMutator,
+	useTicketQuantityForCapacity,
+	useUpdateRelatedTickets,
+	UpdateDatetimeInput,
+} from '@edtrServices/apollo/mutations';
 import useEvent from '@edtrServices/apollo/queries/events/useEvent';
+import { useDatetimeItem } from '@edtrServices/apollo';
 
 const useEditDatetimeModal: FormModal = ({ entityId, entityDbId }) => {
 	const { updateEntity } = useDatetimeMutator(entityId);
 	const { closeEditor } = useFormModal();
 
+	const updateRelatedTickets = useUpdateRelatedTickets(entityId);
+	const ticketQuantityForCapacity = useTicketQuantityForCapacity();
+
 	const { name: eventName = '' } = useEvent() || {};
```

Comment:
is it possible to push some of these defaults into `useEvent`, such that the end result would look:
```
	const { name: eventName } = useEvent();
```
?

Reply:
Why should `useEvent` return an empty object when there is nothing. It's the responsibility of the consumer to handle `undefined`